src/relentless/simulate/simulate.py

```diff
@@ -870,3 +882,28 @@ class AnglePotentialTabulator(PotentialTabulator):
 
     def __init__(self, potentials, num):
         super().__init__(potentials=potentials, start=0.0, stop=numpy.pi, num=num)
+
+
+class DihedralPotentialTabulator(PotentialTabulator):
+    r"""Tabulate one or more dihedral potentials.
+
+    Enables evaluation of energy, force, and derivative at different
+    dihedral values (i.e. :math:`\theta`).
+
+    Parameters
+    ----------
+    potentials : :class:`~relentless.potential.dihedral.DihedralPotential` or array_like
+        The dihedral potential(s) to be tabulated. If array_like, all elements must
+        be :class:`~relentless.potential.dihedral.DihedralPotential`\s.
+    start : float
+        The minimum value of :math:`\theta` at which to tabulate.
+    stop : float
+        The maximum value of :math:`\theta` at which to tabulate.
+    num : int
+        The number of points (value of :math:`\theta`) at which to tabulate and
+        evaluate the potential.
+
+    """
+
+    def __init__(self, potentials, num):
+        super().__init__(potentials=potentials, start=-numpy.pi, stop=numpy.pi, num=num)
```

Comment:
I wanted to confirm: is this the IUPAC definition of the range of a dihedral angle? You should follow that, and also document the domain of the angle when you define the math of it above. (Didn't scroll back to check, so apologies if it's already there.)

Reply:
Yes. IUPAC defines the dihedral to have an absolute value less than 180 degrees with the sign depending on how the bonds are rotated. 

https://goldbook.iupac.org/terms/view/T06406